docs/search/quickstart.mdx

```diff
@@ -55,8 +55,9 @@ be the name of a column that will function as a row's unique identifier within t
 can just be the name of your table's primary key column.
 
 <Note>
```

Comment:
I would just remove the Note altogether here. There is another mention of this here: https://docs.paradedb.com/search/full-text/index under `key_field` which can be updated with your current comment :)